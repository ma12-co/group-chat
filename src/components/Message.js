/** @format */

import React from 'react'
import styled from 'styled-components'

function Message(props) {
  return (
    <>
      <StyledMessage>
        <StyledMessageUserName>{props.username}</StyledMessageUserName>
        <StyledMessageText>{props.text}</StyledMessageText>
      </StyledMessage>
    </>
  )
}

export default Message

const StyledMessage = styled.div`
  margin: 15px;
  border-bottom: 1px solid lightgray;
  padding: 10px;
`

const StyledMessageUserName = styled.div`
  font-size: 11px;
  color: var(--main-text-color);
  opacity: 0.9;
  margin-bottom: 6px;
`

const StyledMessageText = styled.div`
  color: var(--secondary-text-color);
  font-weight: 400;
  display: inline;
`
