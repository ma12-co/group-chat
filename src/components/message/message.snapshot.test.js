/** @format */

import React from 'react'
import {create} from 'react-test-renderer'
import Message from './Message'

describe('Message component', () => {
  test('it matches the snapshot', () => {
    const component = create(<Message />)
    expect(component.toJSON()).toMatchSnapshot()
  })
})
