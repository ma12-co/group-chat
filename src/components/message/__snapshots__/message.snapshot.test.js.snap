// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Message component it matches the snapshot 1`] = `
<div
  className="sc-bdVaJa eTqxec"
>
  <div
    className=" sb-avatar"
    style={
      Object {
        "borderRadius": "100%",
        "display": "inline-block",
        "fontFamily": "Helvetica, Arial, sans-serif",
        "height": "40px",
        "verticalAlign": "middle",
        "width": "40px",
      }
    }
  >
    <div
      className=" sb-avatar__text"
      style={
        Object {
          "background": undefined,
          "borderRadius": "100%",
          "color": "#FFF",
          "height": "40px",
          "lineHeight": "initial",
          "textAlign": "center",
          "textTransform": "uppercase",
          "width": "40px",
        }
      }
    >
      <div
        style={
          Object {
            "display": "table",
            "height": "100%",
            "tableLayout": "fixed",
            "width": "100%",
          }
        }
      >
        <span
          style={
            Object {
              "display": "table-cell",
              "fontSize": "100%",
              "verticalAlign": "middle",
              "whiteSpace": "nowrap",
            }
          }
        >
          <span />
        </span>
      </div>
    </div>
  </div>
  <div
    className="sc-bwzfXH dqogVs"
  >
    <div
      className="sc-htpNat iiFJB"
    />
    <div
      className="sc-bxivhb dtsayX"
    />
  </div>
</div>
`;
